catalog/core/catalog-core-solr/src/main/java/ddf/catalog/source/solr/DynamicSchemaResolver.java

```diff
@@ -430,7 +431,31 @@ private String createCenterPoint(List<Serializable> values) {
       return null;
     }
 
-    return centerPoint.getY() + "," + centerPoint.getX();
+    return normalize(centerPoint);
+  }
+
+  /**
+   * Normalizes point for coordinate boundary wrapping. Truncates coordinates to a reasonable
+   * precision.
+   *
+   * @return Solr formatted coordinate
+   */
+  private String normalize(Point center) {
+    return DistanceUtils.normLatDEG(truncate(center.getY()))
```

Comment:
interesting function... considers going past the top of the world as if you turned back around and walked down.

Reply:
I realized I was starting to write a crappy version of it and then remembered Spatial4j would have the function I needed.  Thankfully it was public and already available to the Solr provider classes.